# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/res/forms.css
# Commit: 2684fdcd6fbf
# Full Hash: 2684fdcd6fbff5abe54a3e1c018e7188abb4076c
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-11-26 09:34:48
# Regressor Bug: 471015
# File Overlap Count: 1
# Description:
#   Bug 471015 - [css-break] Implement <fieldset> fragmentation.  r=TYLin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D53709
# ==============================================================================

diff -r e29eecc5147c -r 2684fdcd6fbf layout/style/res/forms.css
--- a/layout/style/res/forms.css	Mon Nov 25 15:28:50 2019 +0000
+++ b/layout/style/res/forms.css	Mon Nov 25 21:45:22 2019 +0000
@@ -19,6 +19,7 @@
      borders we don't leak out the corners for hit-testing purposes. */
   border-radius: inherit;
   padding: inherit;
+  box-decoration-break: inherit;
   block-size: 100%; /* Need this so percentage block-sizes of kids work right */
   /* Please keep the Multicol/Flex/Grid/Align sections below in sync with
      ::-moz-scrolled-content in ua.css and ::-moz-button-content below. */